pkg/tracing/attributes.go

```diff
@@ -5,7 +5,7 @@ import (
 )
 
 const (
-	TracerName = "kyverno"
+	limit = 256
```

Comment:
Maybe a godoc why this limit is needed would be nice.

Reply:
@MarcelMue I will expose the value in a follow up PR, this should be configurable by end users and we don't want it hardcoded (I wonder if this can be achieved with an env var to let the otel sdk know about it, would be even simpler).
I will add a godoc at this time.